### DISCUSSION TITLE
Lien d'édition non présent

### DISCUSSION THREAD

**Discussion by opetitprestataire** [IGNF-Ma-carte/.github#8] [Cartes narratives] [answered]
Bonjour !

J'ai créé la carte des implantations suivante 
https://macarte.ign.fr/carte/K4OZ7b/
Dans mon équipe :
![carteequipe](https://github.com/user-attachments/assets/7fdc691b-0ecf-4cdb-b0b3-e4db6b5606ae)
Or le lien de modification n'a pas été généré 
![image](https://github.com/user-attachments/assets/76f70fc8-d324-4f2c-931b-100df39eeb83)
![image](https://github.com/user-attachments/assets/8d2faafd-cedc-4539-b31e-bb1e141442f7)

Et je ne peux plus la modifier.
Que se passe-t-il ?

Merci beaucoup

**Comment by Viglino** [comment, marked as answer]
Le lien de modification n'est pas généré automatiquement (dans les équipes),  cliquez sur [Créer un nouveau lien] pour en générer un. 
Vous pouvez aussi changer l'auteur de la carte pour donner l'accès à un utilisateur en particulier.
Pour plus d'informations sur le partage en équipe : 
https://ignf-ma-carte.github.io/chatbot/?q=%23equipe%2FQui%20peut%20voir%20les%20cartes%20d%27une%20%C3%A9quipe
N'hésitez pas à nous re contacter en ças de pb
